graphrag/index/utils/json.py

```diff
@@ -6,6 +6,7 @@
 
 def clean_up_json(json_str: str):
     """Clean up json string."""
+    json_str = json_str[json_str.index('{'):]
```

Comment:
it will raise error in global query

json_str = json_str[json_str.index('{'):]
ValueError: substring not found

Reply:
wrt to your error... no matter how well you write your json parser you'd still encounter error from time to time. 

reason: your model isn't able to output json in a format you require it to.

So far the only solution is reiteration when errors are occured as error will still occur from time to time even if you finetune your gpt4 model. Of course, this is speaking from experience but so far i have not seen any model getting a 100% in humaneval benchs.

Probably a good fix would be reiteration when faced with parsing error and not a fix on the parsing logic.